### ISSUE TITLE
Improve visibility of HTML accessibility

### ISSUE THREAD

**Issue by leggsimon** [kamranahmedse/developer-roadmap#540]
I really love these charts and find them a great steer of things to learn about. It would be nice to see Accessibility a yellow, personal recommendation box for a frontend developer. I’m not sure that knowing about accessibility should be implied as an “optional” extra and more of a requirement of most frontend web development.

**Comment by jsejcksn** [comment]
Of course those of us concerned with accessibility want to improve accessibility's visibility. 😉

But jokes aside, I don't think it's an "optional" extra in the chart: I think it's actually _featured_, as it's one of the few components of learning that HTML is broken into. HTML could have been categorized into many smaller components, but accessibility was chosen as one of four primary components.

Charts like these always have to find a balance of digestibility between lack of detail and information overload, and most of the issues in this repo reflect the individuals' desires to expand their areas of interest. I think we should be glad that accessibility is featured and so close to the top!

**Comment by kamranahmedse** [comment]
Agree with what @jsejcksn said. However, this confusion has been fixed with better legends in the coming release. Thank you for reporting! 🙏
